src/Security/SecurityConfigUpdater.php

```diff
@@ -43,7 +46,7 @@ public function updateForUserClass(string $yamlSource, UserClassConfiguration $u
         return $contents;
     }
 
-    public function updateForAuthenticator(string $yamlSource, string $firewallName, $chosenEntryPoint, string $authenticatorClass, bool $logoutSetup): string
+    public function updateForAuthenticator(string $yamlSource, string $firewallName, $chosenEntryPoint, string $authenticatorClass, bool $logoutSetup, bool $useSecurity51 = false): string
```

Comment:
Is there a need to have the default `false` value? (given that this class is internal, we don't need to care about BC here)

Reply:
good catch! done..